File: app/src/main/java/com/rahbarbazaar/homadit/android/models/edit_products/Member.java
package com.rahbarbazaar.homadit.android.models.edit_products;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Member {
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("id")
    @Expose
    public String id;
}

File: app/src/main/java/com/rahbarbazaar/homadit/android/models/profile/Member.kt
package com.rahbarbazaar.homadit.android.models.profile

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName



class Member {

    @SerializedName("data")
    @Expose
    var data: List<MemberDetail>? = null
}